From 73d5c12930cb7cf8f4735731101a1840c5c12f6c Mon Sep 17 00:00:00 2001
From: Rachel Lee <rnlee@google.com>
Date: Fri, 28 Oct 2022 10:53:00 -0700
Subject: [PATCH 1/3] Remove unused include Util.h.

Util.h is not used in ComposerServiceWriter so it should not be
included. Otherwise when using ComposerServiceWriter, Util.h is not
found in the local directory.

Bug: 255623263
Test: m
Test: flashall to device

Change-Id: I4ce57b37d6a9a899d114e5ac64d443cfa51d826a
Merged-In: I4ce57b37d6a9a899d114e5ac64d443cfa51d826a
---
 .../hardware/graphics/composer3/ComposerServiceWriter.h        | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/graphics/composer/aidl/include/android/hardware/graphics/composer3/ComposerServiceWriter.h b/graphics/composer/aidl/include/android/hardware/graphics/composer3/ComposerServiceWriter.h
index 34cda6a770..b50b84bce6 100644
--- a/graphics/composer/aidl/include/android/hardware/graphics/composer3/ComposerServiceWriter.h
+++ b/graphics/composer/aidl/include/android/hardware/graphics/composer3/ComposerServiceWriter.h
@@ -18,6 +18,7 @@
 
 #include <aidl/android/hardware/graphics/composer3/CommandResultPayload.h>
 #include <aidl/android/hardware/graphics/composer3/IComposerClient.h>
+#include <android-base/logging.h>
 #include <inttypes.h>
 #include <string.h>
 
@@ -26,8 +27,6 @@
 #include <memory>
 #include <vector>
 
-#include "Util.h"
-
 namespace aidl::android::hardware::graphics::composer3::impl {
 
 class ComposerServiceWriter {
-- 
2.34.1

